src/client/Sidebar.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';  // needed?
+import axios from 'axios';
+
+export default class Sidebar extends Component { // need default here?
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: null,
+    };
+  }
+
+  componentWillMount() {
+    axios.get('/api/user/fetch')
```

Comment:
componentWillMount needs to call the `fetchUser` function in the actions folder. Write `fetchUser` for redux